### PR TITLE
Update Webpack, Change Test Suite

### PR DESCRIPTION
**Pull request by KarmicXKoala**
Updates to current version of webpack, uses vue/test-utils + mocha + jsdom for testing instead of karma-mocha + PhantomJS.

All tests have been updated to use jest/expect instead of assert.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    presets: ['env']
+}
```
Comment (dankuck) [inline .babelrc:3]: What does this file do?

Comment (dankuck) [reply .babelrc:3]: Tells babel which version of js to compile down to. Thanks!

**Review by dankuck — APPROVED** [review]
Great! 🐯

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "Easy-to-use HTML tables in Vue.js",
   "main": "dist/NovaTable.js",
   "scripts": {
-    "test": "karma start karma.conf.js"
+    "test": "mocha-webpack --webpack-config webpack.config.js --require test/setup.js test/**/*.spec.js"
   },
```
Comment (dankuck) [inline package.json]: So, I'm a little stuck on how to build the distribution files now. Can you add a `scripts` line to do that for us?

Comment (KarmicXKoala) [reply package.json]: Sure. Will commit updates in the next day or so

**Comment by KarmicXKoala** [comment]
Added the hook. You can run tests with `npm test`, and can build with `npm run build`

**Pull request merged**